airbyte-commons/src/main/resources/log4j2.xml

```diff
@@ -2,40 +2,69 @@
 <Configuration status="INFO">
     <Properties>
 
-        <!-- The following patterns mask the string apikey=<string> to apikey=***** to prevent secrets leaking. -->
-        <Property name="default-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %highlight{%p} %C{1.}(%M):%L - %X - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
-        <!-- The following patter log the application name in the beginning of the line, then a regular log line after a - separator. If the application name is not present in the context, the separator is removed -->
-        <Property name="docker-worker-file-pattern">%replace{%X{log_source} - }{^ - }{}%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %p (%X{job_root}) %C{1}(%M):%L - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
```

Comment:
what happened to these changes? no longer applicable?

Reply:
I think the separation of these was just a mistake that grew over time. There doesn't need to be a difference between default, docker, and cloud with the new settings.  https://airbytehq.slack.com/archives/C0229LM09T4/p1639007638080200 covers the end result and the default pattern works the same way for non-job logs.